dev-python/uhi/uhi-0.4.0.ebuild

```diff
@@ -0,0 +1,27 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..11} )
+DISTUTILS_USE_PEP517=hatchling
+export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}
```

Comment:
`export` should not be necessary here

Reply:
hmm, maybe I am wrong then.

Reply:
It worked without the export, maybe it is just copied from the docs https://projects.gentoo.org/python/guide/distutils.html or other ebuilds with the export.